website/www/site/content/en/contribute/release-guide.md

```diff
@@ -1126,21 +1126,28 @@ Verify that:
 * Images with *latest* tag are pointing to current release by confirming 
   1. Digest of the image with *latest* tag is the same as the one with {RELEASE} tag.
 
+### Merge website pull request
+
+Merge the website pull request to [list the release](/get-started/downloads/), publish the [Python API reference manual](https://beam.apache.org/releases/pydoc/), the [Java API reference manual](https://beam.apache.org/releases/javadoc/) and Blogpost created earlier.
```

Comment:
Shouldn't this be pull request**s** (plural)?

Reply:
Pushed a commit tweaking this section, PTAL